src/client/pages/Course.tsx

```diff
@@ -40,22 +40,17 @@ export default function Course() {
       <div className="curricula-container">
         {course && (
           <>
-            <Typography
-              style={{
-                textAlign: "center",
-                color: "white",
-                fontWeight: "normal",
-                fontSize: "65px",
-                paddingBottom: "3%",
-              }}
-            >
-              {course.title}
+            <Typography variant="h2" style={{ color: getColor(View.COURSES) }}>
+              {course.title.toUpperCase()}
             </Typography>
 
             {/* Provides links to all the topics offered */}
             <ListSelection tiers={course.tiers} view={View.COURSES} courseId={course.id} />
           </>
         )}
+        <Button style={{ backgroundColor: getColor(View.COURSES) }}>
+          <Link href="/courses">BACK</Link>
```

Comment:
Use `react-router-dom`'s `<Link>` component, and then use the `to` prop instead of `href`.

Reply:
(this applies to all `Link`s)

Reply:
but this way I get to customize MUI Link in the theme